### PR TITLE
[WIP] Add static pages for 404, 422 and 500 errors.

### PR DESCRIPTION
**Pull request by alexwllms**
@jongilbraith @lshepstone

### CODE REVIEW

File: public/404.html [hunk 1]
```diff
@@ -1,67 +1,136 @@
...
...eWS27+7Qrp8dOsCQr68B23Hruls2EeaMeHy2+k+C/RJW7KuEXple4L3iegWkOUU9xzuj+VuUFbNO3Hwt16o6mfdE4+4p6XjdMPtNJQwTtlxP3IxyC6UBVIRPTZ37gbke2uG3lax8A3O6l4bJGB6sPPD+gqHfdkM9JJWJw6akOsespiR2E8dgS/AItUwbseL2Z4QsdnN4cRgerFRyO159VBDtkmKUrZZmTNYJjUPNNA47wUDiJnmnyE0IupQThXaqhESU23dXf+dZhFwLAQHi/n7HHHvtqQiWfjk/rk8xBwMka2ffFcrb0nriSi1Nu6uy0Z4LqAUhppxiGALEVA0SgkP6rrfPFBPjug6EmVqCTAvHGGhT5fsR2tjYdw5z+EIEhJOH1Tp0rpp4lrVYjpBJzhx3DbFcn905nhW5Isqt06Or5bR/158iLs56PXFlnX1urHs6vZSDSAVGPeVqPZWxSsJ+I5R+EiMXeIeaue82GRBDgLge13bmsbSde04wjHIjKDNEgiqiIY6FEtaPldOomdvmNe3mBQ5q4tiJyp1jkvaU4G4o4qKZm2Ax69wn4gDf6b0Z+qYceVVTIToi868z2B4fM3yY+DufMn9ixnNdeU6uZ+4deUtsHMr5cALZFp73T7Xjx4R995gBFKwYYz80nlMRnKk59r+FyMvkDt4KnvsPcV6aOX5z+OMds/wnT59RQMnV+UTNu7Lzm5wnY/L6Dp9/WznFzF2T4/LvoX47/PmjlemniHJNW26n8muJ7buzdTBy3kfUw/TBzVQ/DMbObRaXK/ZJOMYlcrm3fZZifIZ0V6hvYuRmnvtg++bOlvt0+JjxbMZ1TL6iac+7idxXvrwTKOfCYwQex8lxJ/h97jFi+7HQuL+fwZA5CZB1q+bn+H2KvrhyD9bZ+s7x+1nnXaSsyol3PaR4j23HamKOPzjkZGRqvBW/WvmqQX2MLvgyNrZ2Xrvm9p1LhwSee5KQC9te56LxUGYV8nBkzxsZjuNrNWj/k12k3nP1oq3XseyhXPf2Y0je11Mk8o/qsRosEIftexSWnw/KLwZyfLLv2A/e8RSQVynpQ2I9Nx7d/yBxLFjZKke5e8eGt6RyY/b5w8Dxcj/TBg8AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAWAb+DwIMlYlovM2EAAAAAElFTkSuQmCC" />
```
Comment (alexwllms) [inline public/404.html]: Done.

**Comment by benlovell** [comment]
:ship:

**Comment by alexwllms** [comment]
@benbarnett can you review the `<head>` and see if there is anything superfluous that can be removed, that I have missed.

File: public/500.html [hunk 2]
```diff
@@ -1,66 +1,135 @@
...
+  <div class="l-constrained">
+    <div class="mas-logo">
+      <img class="mas-logo__img" alt="the Money Advice Service logo" src="mas_logo.png" />
     </div>
-    <p>If you are the application owner check the logs for more information.</p>
   </div>
+</header>
+
+<div class="l-constrained l-error">
+  <h1>We're sorry. The server has stopped responding.</h1>
```
Comment (benbarnett) [inline public/500.html]: How about something more generic? `We're sorry, but something's gone wrong.`

Comment (alexwllms) [inline public/500.html]: :+1:

**Pull request merged**